### DISCUSSION TITLE
I can test things on slurm if that help

### DISCUSSION THREAD

**Discussion by statquant** [wlandau/crew.cluster#42] [General]
Hello, I’ve seen mentions that things are untested on slurm. I am happy to help testing things if that’s remotely useful.

**Comment by wlandau** [comment]
That would be really helpful, thank you!

**Reply by statquant** [reply]
So is that fine if I come back to this discussion with code I try to get suggestions ? I happy to write a Rmd you can add to docs later on.

**Reply by wlandau** [reply]
Whatever works for you.

**Comment by wlandau** [comment]
I'll reopen this discussion if/when there is more to say.
